vault/core.go

```diff
@@ -1027,6 +1038,15 @@ func NewCore(conf *CoreConfig) (*Core, error) {
 		c.customListenerHeader.Store(([]*ListenerCustomHeaders)(nil))
 	}
 
+	logRequestsLevel := log.LevelFromString(conf.RawConfig.LogRequestsLevel)
+	c.logRequestsLevel = uberAtomic.NewInt32(0)
+	switch {
+	case logRequestsLevel > log.NoLevel && logRequestsLevel < log.Off:
+		c.logRequestsLevel.Store(int32(logRequestsLevel))
+	default:
+		c.logger.Warn("invalid log_requests_level", "level", conf.RawConfig.LogRequestsLevel)
```

Comment:
I think this means that if log_requests_level is unspecified, we'll log a warning, which I'd prefer to avoid.